src/documents/tutorial-userInterfaceOptions/WorkingWithUserInterfaceOptions.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Working with UI Options
+layout: default
+category: Tutorials
+---
+
+Once User Interface Options or a Preferences Editor is incorporated into your website (see the [User Interface Options tutorial](./UserInterfaceOptions.md) for installation procedure), it's important to ensure that the site transforms and responds correctly.
+
+This documentation assumes you are somewhat comfortable with writing and editing `CSS`.
+
+## Contrast Themes
+
+UI Options adds support for high contrast themes by transforming the colours on your website. Since contrast themes transform almost all aspects of the webpage, it's probable there will be some styling issues initially. This section aims to help you fix some common issues.
+
+
+UI Options transforms the user interface into one of four high contrast or one low contrast colour combinations:
+
+* high contrast
+    * black on white
+    * white on black
+    * black on yellow
+    * yellow on black
+* low contrast
+    * light grey on dark grey
+
+The themes attempt to remove any other colours from the interface, along with gradients, shadows, background images, and other styling that may reduce legibility. This type of contrast interface can be extremely helpful for users with different forms vision needs.
+
+UI Options applies contrast themes by adding a special class to the `<body>` element of the document and providing styles that are scoped to that class. You will likely need to create a few more special classes, scoped to the theme, for some areas of your site.
+
+To help you in creating a website that works well with UI Options, here are a few basic guidelines:
+
+* [Make cosmetic graphics background images in CSS; create contrast versions; and use CSS to swap in the contrast versions when the themes are activated.](./UsingImagesInContrastModes.md)
+* [Improve clarity and legibility by removing gradients, shadows, and other styling effects.](./StyleEffectsAndLegibilityInContrastModes.md)
+* [Double-check use of CSS "content" property](./ContentPropertyHighContrast.md)
+
+## How to define special styles for contrast themes
+
+If something doesn't transform into contrast mode automatically when using the contrast adjustment, here's how you define styles to address those cases:
+
+### Scope your styles to the theme name
+
+UI Options will add the theme name as a class to the `<body>` element.
+
+* yellow on black: `<body class="fl-theme-uio-yb">`
+* white on black: `<body class="fl-theme-uio-wb">`
+* black on yellow: `<body class="fl-theme-uio-by">`
+* black on white: `<body class="fl-theme-uio-bw">`
+* light grey on dark grey: `<body class="fl-theme-uio-lgdg">`
```

Comment:
these classes don't have "uio" in them anymore. So they would just be like "fl-theme-yb".